custom/enikshay/integrations/bets/repeater_generators.py

```diff
@@ -444,14 +449,14 @@ def get_payload(self, repeat_record, user):
             "phone_numbers": user.phone_numbers,
             "email": user.email,
             "dtoLocation": _get_district_location(location),
-            "privateSectorOrgId": location.metadata['private_sector_org_id'],
+            "privateSectorOrgId": location.metadata.get('private_sector_org_id', ''),
             "resource_uri": "",
         }
         user_json['user_data'] = {
             field: user.user_data.get(field, "")
-            for field in self.user_data_fields
+            for field in BETSUserPayloadGenerator.user_data_fields
```

Comment:
I made this a staticmethod so I could access it from the management command, which means `self` is no longer available.